iidm/iidm-reducer/src/main/java/com/powsybl/iidm/reducer/DefaultNetworkReducer.java

```diff
@@ -178,6 +182,57 @@ private Load replaceBranchByLoad(Branch<?> branch, VoltageLevel vl, Terminal ter
         return load;
     }
 
+    private void replaceHvdcLine(HvdcLine hvdcLine, VoltageLevel vl, Terminal terminal, HvdcConverterStation.HvdcType type) {
+        if (type == HvdcConverterStation.HvdcType.VSC) {
```

Comment:
This this the invert and only if the VSC has the boolean voltageRegulatorOn to true. LCC are always replaced by a load.

Reply:
Changes are done (if I understood right).